### DISCUSSION TITLE
Wordpress org following calypso?

### DISCUSSION THREAD

**Discussion by uhlhosting** [Automattic/wp-calypso#81348] [Q&A] [answered]
Hi,

Is nice that you guys managed to revamp the old wordpress.com. Yet to me is not seems fair to leave behind wordpress.org. So are there plans to revamp wordpress org into something similar to calypso? Or that will never happend?

**Comment by lynnbgriffith** [comment, marked as answer]
There are some early efforts to redesign the wp-admin experience to include many of the refinements that have been made as part of the Gutenberg project for other aspects of the admin:
https://make.wordpress.org/core/2023/07/12/admin-design/
A lot of this work has started in the recent versions of the site editor in the Gutenberg plugin and the WP.org core releases, and subsequent releases will just broaden the scope of those new approaches to more and more use cases. As this happens, we're thinking about how Calypso can evolve to overlap less with these newer approaches in wp-admin. As Gutenberg matured, for example, Calypso's own post editor was removed. If a modern page and post list is added in wp-admin, we'll evaluate then how we want to approach those features in Calypso.
